docs/rebuild_admin.rst

```diff
@@ -0,0 +1,407 @@
+
+First, log on to the *Application Server* via the console and edit the file 
+``/etc/ssh/sshd_config``, deleting the line:
+
+.. code-block:: none
+
+  PasswordAuthentication no                                         
+
+Then, restart ``sshd`` using the command ``sudo service sshd restart``.
+
+Finally, check the file ``/etc/networking/interfaces``, and note down the network
+settings for the default Ethernet interface. You'll need them in the next step.
+
+Repeat the process above for the *Monitor Server*, making sure to note down its
+network settings as well.
+
+Enabling access from the new *Admin Workstation*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+From the *Admin Workstation*, open a terminal and copy the *Admin Workstation's* 
+SSH public key to the servers, substituting the appropriate values for the username
+and server IP addresses in the commands below and entering the admin account's 
+password when prompted:
+
+.. code:: sh
+
+  ssh-copy-id <admin-username>@<application-server-ip>
+  ssh-copy-id <admin-username>@<monitor-server-ip>
+  
+Next, create a file ``~/.ssh/config`` with contents as below, again substituting
+the appropriate values for your servers:
+
+.. code-block:: none
+ 
+  Host app
+    User <admin-username>
+    Hostname <application-server-ip>
+    ProxyCommand none
+
+  Host mon
+    User <admin-username>
+    Hostname <monitor-server-ip>
+    ProxyCommand none
+
+
+Finally, test direct SSH access from the terminal, using the commands ``ssh app`` and 
+``ssh mon``. It should be possible to connect without entering a password.
+
+Step 4: Retrieve SecureDrop configuration info from the servers
+===============================================================
+
+In addition to the account and networking information retrieved from the servers
+so far, you'll need to retrieve the following files and info:
+
+ - Tor hidden service URLs and tokens
+ - GPG *Submission Public Key* and OSSEC public key
+ - OSSEC alert configuration details
+ - (Optional) HTTPS configuration details
+
+Retrieve Onion hidden service info
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to connect to the Tor hidden services used by your instance, you will
+need to copy their details to the *Admin Workstation*. To do so, run the following
+commands from a Terminal window:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop/install_files/ansible-base
+ echo "HidServAuth $(ssh app sudo cat /var/lib/tor/services/journalist/hostname)" \
+    > app-journalist-aths
+ echo "$(ssh app sudo cat /var/lib/tor/services/journalist/hostname)" > app-source-ths
+
+If your instance was set up to use SSH over TOR, you'll also need to copy over the details 
+of the SSH proxy hidden services, by running the following commands:
+
+.. code:: sh
+
+ echo "HidServAuth $(ssh app sudo cat /var/lib/tor/services/ssh/hostname)" > app-ssh-aths
+ echo "HidServAuth $(ssh mon sudo cat /var/lib/tor/services/mon/hostname)" > mon-ssh-aths
```

Comment:
Should be `/var/lib/tor/services/ssh/hostname`, not `/var/lib/tor/services/mon/hostname`